packages/@n8n/task-runner-python/src/main.py

```diff
@@ -3,35 +3,54 @@
 import os
 import sys
 
-from .constants import ENV_TASK_BROKER_URI, ENV_GRANT_TOKEN, DEFAULT_TASK_BROKER_URI
-from .task_runner import TaskRunner
-
-logging.basicConfig(
-    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+os.environ["WEBSOCKETS_MAX_LOG_SIZE"] = "256"
+
+from .constants import (
+    DEFAULT_MAX_CONCURRENCY,
+    DEFAULT_TASK_TIMEOUT,
+    ENV_MAX_CONCURRENCY,
+    ENV_MAX_PAYLOAD_SIZE,
+    ENV_TASK_BROKER_URI,
+    ENV_GRANT_TOKEN,
+    DEFAULT_TASK_BROKER_URI,
+    DEFAULT_MAX_PAYLOAD_SIZE,
+    ENV_TASK_TIMEOUT,
 )
-
-logger = logging.getLogger(__name__)
+from .logs import setup_logging
+from .task_runner import TaskRunner, TaskRunnerOpts
 
 
 async def main():
-    task_broker_uri = os.getenv(ENV_TASK_BROKER_URI, DEFAULT_TASK_BROKER_URI)
-    grant_token = os.getenv(ENV_GRANT_TOKEN, "")
+    setup_logging()
+    logger = logging.getLogger(__name__)
+
+    logger.info("Starting runner...")
 
-    if not grant_token:
+    grant_token = os.getenv(ENV_GRANT_TOKEN)
+
+    if grant_token is None:
         logger.error(f"{ENV_GRANT_TOKEN} environment variable is required")
         sys.exit(1)
 
-    runner = TaskRunner(
-        task_broker_uri=task_broker_uri,
-        grant_token=grant_token,
+    assert grant_token is not None
+
+    opts = TaskRunnerOpts(
+        grant_token,
+        os.getenv(ENV_TASK_BROKER_URI, DEFAULT_TASK_BROKER_URI),
+        int(os.getenv(ENV_MAX_CONCURRENCY, DEFAULT_MAX_CONCURRENCY)),
+        int(os.getenv(ENV_MAX_PAYLOAD_SIZE, DEFAULT_MAX_PAYLOAD_SIZE)),
+        int(os.getenv(ENV_TASK_TIMEOUT, DEFAULT_TASK_TIMEOUT)),
```

Comment:
Looked into `pydantic` to handle this more elegantly, but it brings in almost 10 dependencies, which I'm not willing to accept only for a minor convenience.